stacks-signer/src/lib.rs

```diff
@@ -34,3 +34,6 @@ pub mod runloop;
 pub mod signer;
 /// The state module for the signer
 pub mod signerdb;
+
+/// The monitoring server for the signer
```

Comment:
I think if you put the clippy allows above this 'pub mod monitoring' you can remove the duplicated '#[allow(unused_variables)]' everywhere.  (Not 100% sure, but worth a try)

Reply:
Yeah it's messy having all of those `allow`s, but I kinda prefer it vs a top-level flag so that I can catch any mistakes in feature-flagged code.